src/components/EventCard/index.js

```diff
@@ -67,6 +79,8 @@ EventCard.propTypes = {
   totalBOT: PropTypes.number,
   endTime: PropTypes.string,
   buttonText: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/no-typos
+  intl: intlShape.isRequired,
```

Comment:
do we have to name it `intl`? or can we name it `localize`?

I am not liking how we have to disable a linting rule for this in all the places.

Reply:
I tried to name it localize or something else, but turned out the typo is only for `intlShape`.